standard-app/templates/configs/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+# Default configMap
+{{- if .Values.configMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-configmap
```

Comment:
`.Chart.Name` would be substituted with `standard-app` name.
Use `.Release.Name` instead.

Reply:
name changed from `.Chart.Name` to `.Release.Name` for default configMap